### PR TITLE
Save pets level on reload.

### PR DESCRIPTION
**Pull request by hostedposted**
## Description
I save pets level in a array in local storage and push an array with id then level of pet for all pets.

Then when you reload a for each loop is done and the pets levels are changed to the ones in local storage,

### CODE REVIEW

**Comment by ArcerionDev** [comment]
Like the idea, but if someone were to switch their pet team, it'd change the level of the new pet.

**Comment by hostedposted** [comment]
Ok.

**Comment by ChickenOnYT** [comment]
Then ask them to fix it.

**Review by ChickenOnYT — CHANGES_REQUESTED** [review]
Fix what Arc said.

**Comment by hostedposted** [comment]
Oops did not mean to click that.

File: willsCheatMenu/src/hacks/pets.ts [hunk 1]
```diff
@@ -81,36 +81,48 @@ new Hack(category.pets, "Add Pet", "Adds a pet from a list.").setClick(async ()
...
-})
+new Hack(category.pets, "Uncap pet level (client side only)", "Change your pet's level to anything, even over 100.").setClick(async () => {
+	let petTeam = _.player.kennel.petTeam.slice(0)
+	petTeam.shift()
+	let filter = []
+	petTeam.forEach(e => {
+	if(e){
+	filter.push(e)
+	}
+	})
```
Comment (PatheticMustan) [inline willsCheatMenu/src/hacks/pets.ts:92]: what the hell is this

File: willsCheatMenu/src/index.ts [hunk 2]
```diff
@@ -140,6 +140,17 @@ if (localStorage.getItem("level")) {
 	eval(`_.player.getLevel = () => {return ${localStorage.getItem("level")}}`)
 }
 
+if (localStorage.getItem("pets")) {
+	let pets = JSON.parse(localStorage.getItem("pets"))
+	pets.forEach(element => {
+		_.player.kennel.petTeam.forEach(pet => {
+			if (pet.getID() == element[0]) {
+				eval(`_.player.kennel.petTeam[${element[0]+1}].getLevel = () => {return ${element[1]}}`)
```
Comment (PatheticMustan) [inline willsCheatMenu/src/index.ts:148]: I don't know how you managed it but this is one of the worst lines of code I've ever seen

**Comment by PatheticMustan** [comment]
what is even supposed to happen here

**Pull request closed without merging**